Makefile

```diff
@@ -88,7 +88,8 @@ endif
 
 build_dir := $(DIR)/.build/$(os1)-$(arch1)
 
-go_version := $(shell cat .go-version)
+go_version_full := $(shell cat .go-version)
+go_version := $(go_version_full:.0=)
```

Comment:
Will this do the right thing with future version numbers like 2.0.3?

Reply:
It should be ok. If the full version doesn't end in .0, the go_version will be the same as the full version. When go reaches version `2.0.0`, I'm unsure of what the official version number will be. If it is `2.0`, then we're good. If it is `2`, then this won't work. But that's a while off and we can revisit when then.
```
1.14.0 ==> 1.14
1.14.1 ==> 1.14.1
 2.0.3 ==> 2.0.3
 2.0.0 ==> 2.0     (?)
```